app/src/main/java/org/sopt/and/SignUpActivity.kt

```diff
@@ -0,0 +1,225 @@
+    var email by remember { mutableStateOf("") }
+    var password by remember { mutableStateOf("") }
+    var passwordVisible by remember { mutableStateOf(false) }
+    var showErrorSnackbar by remember { mutableStateOf(false) }
+    val snackbarHostState = remember { SnackbarHostState() }
+
+    fun onSignUpClick() {
+        if (!isValidEmail(email) || !isValidPassword(password)) {
+            showErrorSnackbar = true
+            return
+        }
+
+        val intent = Intent(context, SignInActivity::class.java).apply {
+            putExtra(KeyStorage.EMAIL, email)
+            putExtra(KeyStorage.PASSWORD, password)
+        }
+
+        (context as SignUpActivity).setResult(RESULT_OK,intent)
+        context.startActivity(intent)
+    }
+
+    LaunchedEffect(showErrorSnackbar) {
+        if (showErrorSnackbar) {
+            snackbarHostState.showSnackbar(context.getString(R.string.sign_up_failure))
+            showErrorSnackbar = false
+        }
+    }
+
+    Scaffold(
+        modifier = Modifier.fillMaxSize(),
+        topBar = {
+            CenterAlignedTopAppBar(
+                title = { Text(stringResource(R.string.sign_up_top_bar), style = MaterialTheme.typography.titleMedium) },
+                modifier = Modifier
+            )
+        },
+        snackbarHost = { SnackbarHost(snackbarHostState) },
+        content = { innerPadding ->
+            Column(
+                modifier = Modifier
+                    .padding(innerPadding)
+                    .padding(8.dp),
+                verticalArrangement = Arrangement.spacedBy(16.dp)
+            ){
+                Text(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(8.dp),
+                    text = stringResource(R.string.sign_up_title),
+                    style = MaterialTheme.typography.titleLarge.copy(lineHeight = 32.sp),
+                    color = Color.White, fontWeight = FontWeight.Normal,
+                )
+
+                InputField(
+                    modifier = Modifier,
+                    placeholder = stringResource(R.string.sign_up_email_input_example),
+                    value = email,
+                    onValueChange = {email = it})
+                Text(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = 8.dp),
+                    text = stringResource(R.string.sign_up_email_sub_title),
+                    style = MaterialTheme.typography.bodySmall, color = Color.Gray,
+                    fontWeight = FontWeight.Normal
+                )
+
+                PasswordInputField(
+                    Modifier,
+                    stringResource(R.string.sign_up_password_input_example),
+                    password,
+                    { password = it },
+                    passwordVisible,
+                    { passwordVisible = !passwordVisible },
+                )
```

Comment:
InputField 썼던 부분처럼 수정해주세요 ~
```suggestion
                PasswordInputField(
                    modifier = Modifier,
                    placeholder = stringResource(R.string.sign_up_password_input_example),
                    value = password,
                    onValueChange = { password = it },
                    passwordVisible = passwordVisible,
                    onVisibilityChange = { passwordVisible = !passwordVisible },
                )
```